teams/docs-rs.toml

```diff
@@ -27,3 +25,6 @@ discord-name = "#docs-rs"
 
 [[lists]]
 address = "docs-rs@rust-lang.org"
+
+[[lists]]
+address = "help@docs.rs"
```

Comment:
What does this email address do?

Reply:
It was an email that was in the `docs` team that only included misdreavus and onur. I'm just adding it here to redirect any email that could be sent to that address can be sent to the rest of the team instead.

Reply:
Will emails sent to that address get forwarded to `docs-rs@rust-lang.org`? Since misdreavus and onur left it looks like no one has access to that address anymore.

Reply:
I believe it will just create another mailing list that's consists of the the same members as `docs-rs@rust-lang.org`. So this will essentially be an alias for that list.

Reply:
Yes, and TIL we had such mailing list.